doc/rados/operations/upmap.rst

```diff
@@ -1,52 +1,53 @@
 .. _upmap:
 
-Using the pg-upmap
-==================
+Using pg-upmap
+==============
 
-Starting in Luminous v12.2.z there is a new *pg-upmap* exception table
+In Luminous v12.2.z and later releases, there is a *pg-upmap* exception table
 in the OSDMap that allows the cluster to explicitly map specific PGs to
-specific OSDs.  This allows the cluster to fine-tune the data
-distribution to, in most cases, perfectly distributed PGs across OSDs.
+specific OSDs. This allows the cluster to fine-tune the data distribution to,
+in most cases, perfectly distributed PGs across OSDs.
 
-The key caveat to this new mechanism is that it requires that all
-clients understand the new *pg-upmap* structure in the OSDMap.
+However, there is an important caveat when it comes to this new feature: it
+requires all clients to understand the new *pg-upmap* structure in the OSDMap.
 
 Enabling
 --------
 
-New clusters will by default enable the `balancer module`. The cluster must only
-have Luminous (and newer) clients. You can turn the balancer off with:
+By default, new clusters enable the *balancer module*. The cluster cannot have
+any pre-Luminous clients.  To turn off the balancer, run the following command:
```

Comment:
Re-order

To use `pg-upmap`, the cluster cannot have any pre-Luminous clients. Also, by default, new clusters enable the *balancer module*, which uses `pg-upmap` behind the scenes. If you wish to use an alternative balancer, or enter your own custom `pg-upmap` entries, you may wish to turn off the balancer to avoid conflict:

Reply:
I've ingested something very close to this.